docs/installation/linux.md

```diff
@@ -92,6 +92,11 @@ You can also install the RPM without the intermediate step of downloading it:
 ```sh
 sudo yum install https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.17/powershell-6.0.0_alpha.17-1.el7.centos.x86_64.rpm
 ```
+You can also install via the Microsoft Repository
+sudo rpm -Uvh http://packages.microsoft.com/config/rhel/7/packages-microsoft-prod.rpm
```

Comment:
This should have code blocks.  I can add them.  Just ping me if you want me to do it.